plugins/modules/backup_vault_info.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+DOCUMENTATION = r"""
+---
+module: backup_vault_info
+version_added: 6.0.0
+short_description: Describe AWS Backup Vaults.
+description:
+  - Gets info about Backup Vault configuration.
+author:
+  - Gomathi Selvi Srinivasan (@GomathiselviS)
+options:
+  backup_vault_names:
+    type: list
+    elements: str
+    default: []
+    description:
+      - Specifies a list of vault names.
+      - If an empty list is specified, information for the backup vaults in the current region is returned.
+
+extends_documentation_fragment:
+  - amazon.aws.common.modules
+  - amazon.aws.region.modules
+  - amazon.aws.boto3
+"""
+
+EXAMPLES = r"""
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+# Gather information about all backup vaults
+- amazon.aws.backup_vault_info
+
+# Gather information about a particular backup vault
+- amazon.aws.backup_vault_info:
+    backup vault_names:
+      - "arn:aws:backup_vault:us-east-2:123456789012:backup_vault/defaultvault"
+"""
+
+RETURN = r"""
+backup_vault_info:
```

Comment:
We should  probably rename `backup_vault_info` to something like `backup_vaults`. @tremble do you have a better idea?

Reply:
I will change it to backup_vaults, based on how other info modules are implemented.